### PR TITLE
Update session lifetime limits

### PR DESCRIPTION
**Pull request by tamigoodall**
Update session lifetime limit information to clarify how it works

### CODE REVIEW

**Comment by tamigoodall** [comment]
Randy Nasson provided review comments through Slack DM.

**Comment by tamigoodall** [comment]
Content to go live on February 27, 2019.

File: articles/sso/current/configure-session-lifetime-limits.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  - session-lifetime-limits
+contentType:
+  - how-to
+useCase:
+  - integrate-saas-sso
+  - configure-sso
+---
+# Configure Session Lifetime Limits for Single Sign On
+
+You can configure how you want user session timeouts to work for Single Sign On (SSO) with session lifetime limit settings in the dashboard. You can also configure session limits with the [Management API](/api/management/v2#!/Tenants/patch_settings).
```
Comment (erjohnson) [inline articles/sso/current/configure-session-lifetime-limits.md]: > You can configure how user session timeouts work for Single Sign On (SSO) with session lifetime limit settings in the dashboard.

maybe add a link to `${manage_url}/#/tenant/advanced` here as well

File: articles/sso/current/configure-session-lifetime-limits.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+* If the user is active within the 3-day **Inactivity timeout** period, the session lifetime is extended from that point for another 3 days (the duration of the **Inactivity timeout** period).
+
+* If the user is not active when the **Inactivity timeout** limit is reached, the user will automatically be logged out. This type of session extension may continue until the **Require log in after** limit is reached, after which the user will be forced to log in again regardless of activity. 
+
+If you extend existing session lifetime limits, those limits remain unchanged until a user establishes a new session. For example, if a user has an **Inactivity timeout** limit of 1 day and a **Require log in after** limit of 3 days and you increase the limits to 3 days and 7 days respectively, the user will not experience longer session lifetimes until after the existing session ends and they establish a new session.
+
+::: note
+When you reduce existing session lifetime limits, the user session lifetime limits change *immediately* upon the next registered activity. This behaviour allows you to immediately shorten inadvertently long session lifetime limits for security purposes.
```
Comment (erjohnson) [inline articles/sso/current/configure-session-lifetime-limits.md]: Maybe take this out of the note and put it in the "how it works"

**Review by erjohnson — APPROVED** [review]

**Pull request merged**